data/tome_of_beasts_2/monsters.json

```
@@ -11942,7 +11942,7 @@
        {
               "name": "Infernal Swarm",
               "size": "Huge",
-              "type": "fiends",
+              "type": "Fiend",
               "subtype": "devil",
```

```suggestion
        "subtype": "Swarm of Devils",
```

This is the only case I could find where there was a collision using subtype to indicate "swarm-ness", so I followed the pattern that's in the books. I don't think this is a perfect solution, but it will at least store that these are all swarm type creatures in one of their props, and should be relatively quick to change if we come up with a better solution.